### PR TITLE
Verification channges for obs.

### PR DESCRIPTION
**Pull request by michelleharrold**
## DESCRIPTION OF CHANGES: 
The main goal of this PR is to provide documentation via the config_defaults.sh for describing how to verify with user-provided observation files. Five files were changed within for this PR:

	scripts/exregional_run_gridstatvx.sh --> added a check for OBS_DIR existence; exit if it does not exist
	scripts/exregional_run_pointstatvx.sh --> added a check for OBS_DIR existence; exit if it does not exist
	ush/config_defaults.sh --> provided documentation/comments for how to use user-provided observation files
	ush/load_modules_run_task.sh --> changed path to necessary environment specs to match ufs-srweather-app master
	ush/templates/FV3LAM_wflow.xml --> moved if statements for get obs and vx tasks, so it would not print comment divisions between tasks when they are not activated

## TESTS CONDUCTED: 
Three tests were successfully run:
	1)Tested updated path to environment
	2)Tested check for OBS_DIR existence by not properly specifying the OBS_DIR
	3)Tested with user-provided obs

@jwolff-ncar - ready for review!

### CODE REVIEW

File: ush/config_defaults.sh [hunk 1]
```diff
@@ -373,20 +373,86 @@ FCST_LEN_HRS="24"
...
+# {YYYYMMDD}/ccpa.t{HH}z.01h.hrap.conus.gb2, where YYYY is the 4-digit 
+# valid year, MM the 2-digit valid month, DD the 2-digit valid day of 
+# the month, and HH the 2-digit valid hour of the day. In addition, a 
+# caveat is noted for using hourly CCPA data. There is a problem with 
+# the valid time in the metadata for files valid from 19 - 00 UTC (or 
+# files  under the '00' directory). The script to pull the CCPA data 
+# from the NOAA HPSS has an example of how to account for this as well
+# as organizing the data into a more intuitive format: 
+# regional_workflow/scripts/exregional_get_ccpa_files.sh. When a fix
+# is provided, it will be accounted for the exregional_get_ccpa_files.sh
```
Comment (jwolff-ncar) [inline ush/config_defaults.sh]: Very small typo - it will be accounted for "IN" the exregional_get_ccpa_files.sh.

File: ush/config_defaults.sh [hunk 2]
```diff
@@ -373,20 +373,86 @@ FCST_LEN_HRS="24"
...
+# forecast being verified; since MRMS composite reflectivity files do 
+# not always exactly match the valid time, a script, within the main 
+# script to retrieve MRMS data from the NOAA HPSS is used to identify
+# and rename the MRMS composite reflectivity file to match the valid
+# time of the forecast. The script to pull the MRMS data from the NOAA
+# HPSS as well as the organization of data into a more intuitive format
+# is located here: regional_workflow/scripts/exregional_get_mrms_files.sh.
+# The script used to identify the file closest to the valid time is 
+# located here: regional_workflow/scripts/mrms_pull_topofhour.py
+#
```
Comment (jwolff-ncar) [inline ush/config_defaults.sh:432]: Maybe add that these scripts can be used as examples to rename/organize the data? (Similar to what you did above?) Maybe it is sufficiently implied and not necessary...

**Review by jwolff-ncar — COMMENTED** [review]
Looks great to me. Two really small comments added.

**Comment by michelleharrold** [comment]
Jamie -- thanks for the review! I addressed your comments and committed the changes. Thanks!

**Comment by michelleharrold** [comment]
Updated comment -- when double checking everything, I happened to notice that there are two sections for METplus-related variables. In my config_defaults.sh, there is one section around lines 371-450 and another around 962-974. This appears to be repeat or deprecated info. I propose taking out 962-974. Thoughts?

**Comment by jwolff-ncar** [comment]
Looks good to me. I will merge it in! Thanks.

**Pull request merged**